locations/spiders/casino.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+import re
+
+class CasinoGeojsonPointItem(GeojsonPointItem):
+    addr_2_housenumber = scrapy.Field()
```

Comment:
Instead of creating your own item, please add extra fields to the existing `extras` property.